### DISCUSSION TITLE
Issue with SLEAP Training: "Insufficient system resources exist to complete requested service"

### DISCUSSION THREAD

**Discussion by jckraus14** [talmolab/sleap#2037] [Help!]
I'm attempting to train a SLEAP model using labeled frames in SLEAP 1.3.3. The files for SLEAP are being stored on a server and trained with a NVIDIA GPU cluster. 

While training a single animal model with default parameters preset, I am experiencing an error "Insufficient system resources exist to complete request service." I have included the error and redacted personal information in the image below.

![Screen Shot 2024-11-26 at 3 43 15 PM](https://github.com/user-attachments/assets/43f0a381-a2bf-49cf-8e88-6a916656ecae)

Any support would be greatly appreciated. Thank you!

**Comment by talmo** [comment]
Hi @jckraus14,

Any chance that your `Y:` drive is full or has weird permissions?

You could try copying the project over to a local drive on the server if that's a network mounted storage system.

Cheers,

Talmo

**Reply by jckraus14** [reply]
Hi, I copied the project over to a local drive, and it worked! Thank you!
